modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache

```diff
@@ -75,7 +75,9 @@ static void (^reachabilityChangeBlock)(int);
         [config.loggingFileHanlder writeData:[message dataUsingEncoding:NSUTF8StringEncoding]];
     }
 
-    NSLog(@"%@", message);
+    if ([[{{classPrefix}}Configuration sharedConfig] debug]){
```

Comment:
I assumed it was designed so that you could write logs to a file and not to the console output. I thought the purpose of the debug flag was just for console output, but you could set that off and still set a loggingFile to have the output be written there. Is this not correct?

> On Sep 17, 2015, at 7:13 PM, wing328 <notifications@github.com> wrote:
> 
> In modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache:
> 
> > @@ -75,7 +75,9 @@ static void (^reachabilityChangeBlock)(int);
> >          [config.loggingFileHanlder writeData:[message dataUsingEncoding:NSUTF8StringEncoding]];
> >      }
> >  
> > -    NSLog(@"%@", message);
> > +    if ([[{{classPrefix}}Configuration sharedConfig] debug]){
> @mad102190 would it be better to check the debug flag at the beginning of the method (logResponse) instead and exit the method if debug is false?
> 
> —
> Reply to this email directly or view it on GitHub.
>